### PR TITLE
187 thanks page formatting

### PR DESCRIPTION
**Pull request by shakalee14**

### CODE REVIEW

**Comment by shakalee14** [comment]
for #187

**Comment by shakalee14** [comment]
I also tried to make email language look more like what's on prod, using the following example:
http://petitions.moveon.org/sign/keep-hearst-elementary
vs.
localhost:8080/#/sign/keep-hearst-elementary

File: src/components/thanks.js [hunk 1]
```diff
@@ -43,11 +42,6 @@ class Thanks extends React.Component {
     petitionActions.recordShareClick(this.props.petition, medium, source, this.props.user)
   }
 
-  shareLink(evt) {
```
Comment (schuyler1d) [inline src/components/thanks.js]: I know this isn't part of prod, but can we keep this -- it drives me crazy that I can't get a petition link from the thanks page.

File: src/components/thanks.js [hunk 2]
```diff
@@ -137,14 +130,14 @@ Thanks!
...
-                                                   petition.summary,
+                                                   petition.title,
                                                    this.isCreator,
                                                    shareOpts,
                                                    petition.target,
                                                    `${petition._links.url}?source=${this.state.pre}.em.__TYPE__&${this.trackingParams}`
                                                    )
-    const copyPasteMessage = `Subject: ${petition.summary}\n\n${mailToMessage.replace('__TYPE__', 'cp')}`
-    const mailtoMessage = `mailto:?subject=${encodeURIComponent(petition.summary)}&body=${encodeURIComponent(mailToMessage.replace('__TYPE__', 'mt'))}`
+    const copyPasteMessage = `Subject: ${petition.title}\n\n${mailToMessage.replace('__TYPE__', 'cp')}`
```
Comment (schuyler1d) [inline src/components/thanks.js]: this is definitely correct.

File: src/components/thanks.js [hunk 3]
```diff
@@ -137,14 +130,14 @@ Thanks!
     }
 
     const mailToMessage = this.generateMailMessage(textDescription.textContent,
-                                                   petition.summary,
+                                                   petition.title,
```
Comment (schuyler1d) [inline src/components/thanks.js]: You're right that we need the title as well as the summary, but it shouldn't universally replace it.  You are covering this case:
https://github.com/MoveOnOrg/motrunk1/blob/soak/www/pac/signon/share.mc#L155

but see the other cases above that

**Pull request merged**